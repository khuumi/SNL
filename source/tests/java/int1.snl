start int_stage:
	do show to 0
	do show to 123412
	do show to -43
done
