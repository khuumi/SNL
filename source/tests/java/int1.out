0
123412
-43
